src/web/src/components/Posts/Timeline.tsx

```diff
@@ -42,7 +42,7 @@ const Timeline = ({ pages, nextPage }: Props) => {
   // Iterate over all the pages (an array of arrays) and then convert all post
   // elements to <Post>
   const postsTimeline = pages.map((page: Post[]): Array<ReactElement> | ReactElement =>
-    page.map(({ id, url }: Post) => <PostComponent postUrl={`${telescopeUrl}${url}`} key={id} />)
+    page.map(({ id }: Post) => <PostComponent postUrl={`${postsServiceUrl}/${id}`} key={id} />)
```

Comment:
Can you not use the `url` bare here, since we're now returning an absolute URL?  We shouldn't need to build it ourselves.